docs/backends/exasol.qmd

```diff
@@ -0,0 +1,96 @@
+# Exasol
+
+[https://www.exasol.com](https://www.exasol.com)
+
+## Install
+
+Install Ibis and dependencies for the Exasol backend:
+
+::: {.panel-tabset}
+
+## `pip`
+
+Install with the `exasol` extra:
+
+```{.bash}
+pip install 'ibis-framework[exasol]'
+```
+
+And connect:
+
+```{.python}
+import ibis
+
+con = ibis.exasol.connect(...)  # <1>
+```
+
+1. Adjust connection parameters as needed.
+
+## `conda`
+
+Install for Exasol:
+
+```{.bash}
+conda install -c conda-forge ibis-exasol
```

Comment:
@Nicoretti -- I will try to remember to add this to the `conda-forge` recipe when we do our next release, but you should feel free to ping me to remind me.

Also, if you can add `sqlalchemy-exasol` to `conda-forge` that will greatly improve the turnaround time on getting `ibis-exasol` added.

Reply:
> @Nicoretti -- I will try to remember to add this to the `conda-forge` recipe when we do our next release, but you should feel free to ping me to remind me.

Removing it from the docs would be a possibility too. I just used another backend documentation as template, wasn't aware about all implications. But if we want to keep it, I can over to create a ticket as reminder.

> Also, if you can add `sqlalchemy-exasol` to `conda-forge` that will greatly improve the turnaround time on getting `ibis-exasol` added.

Previously, `sqlalchemy-exasol` was available as a `conda-forge` package, but it was discontinued, possibly due to being outdated and underused. I need to determine if we should reintroduce it. Given that we now have a concrete use case with Ibis, it seems more likely that its reintroduction would be accepted.

Reply:
I can help maintain the feedstock (what's one more on a big pile :grin:?) -- I think it's a good idea to have it available on conda-forge, if only so that `exasol` isn't the only backend that you can't install with conda/mamba.

Reply:
recipe PR here: https://github.com/conda-forge/staged-recipes/pull/24777

Reply:
the underscore threw me when I was looking before.  looks like the conda-forge package is available and up-to-date, so that part is taken care of.